File: 프로그래머스/lv2.JadenCase 문자열 만들기.js
// Level2: JadenCase 문자열 만들기
function solution(s) {
    let answer = '';
    s = s.toLowerCase();
    s = s.split("");
    s.map((e,i) => {
        if(i === 0 || s[i-1] === ' '){
            answer += e.toUpperCase();
        }
        else answer += e;
    })
    return answer;
}

File: 프로그래머스/lv2.JadenCase 문자열 만들기.py
# Level2: JadenCase 문자열 만들기
# def solution(s):
#     answer = ''
#     answer += s[0].upper()
#     for i in range(1,len(s)):
#         if s[i-1] == " ":
#             answer += s[i].upper()
#         else :
#             answer += s[i].lower()
#     return answer

def solution(s):
    s = s.lower().split(" ")
    answer = ''
    for word in s:
        answer += (word.capitalize() + " ")
    return answer[:-1]